SPEC.md

```diff
@@ -255,3 +264,4 @@ IPAM plugin examples:
 
 ## Well-known Error Codes
 - `1` - Incompatible CNI version
+- `2` - Unknown network configuration passed to plugin.
```

Comment:
It would be good to qualify "unknown" a bit more. An example might be helpful somewhere.